### PR TITLE
Update pickup-locations.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Modify Response for the pickup location with no result found, Explain Search Term attribute field.

## Purpose of this pull request

This pull request (PR) contains response for the pickup location with no result found. Search term in the pickup location query definition updated.You must have to pass two-letter country code to get actual result otherwise it will throws an error.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/graphql/queries/pickup-locations.html

### CODE REVIEW

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Because `pageSize` is set to 1 and distance by `distance` is applied, result wil
 ```
 
 **Response:**
+For the available Pick Up location, The Response would be looks like.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:75]:
```suggestion
For the available Pickup location, the response would look like:
```

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 2]
```diff
@@ -106,6 +107,24 @@ Because `pageSize` is set to 1 and distance by `distance` is applied, result wil
 }
 ```
 
+If Pickup location will not available for the given search term, The Response will be emtpy,
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:110]:
```suggestion
If the Pickup location is not available for the given search term, the response will be empty:
```

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 3]
```diff
@@ -125,7 +144,7 @@ Use the `AreaInput` to apply filtration by distance. All attributes are required
 Attribute | Data type | Description
 --- | --- | ---
 `radius` | Int! | The radius to the search, in kilometers.
-`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  For example, `Austin:US`.
+`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid Search Term, `Texas:US`|`Austin:US`|`78740:US`. After Colon, You must required to pass Two-letter Country code in Upper case.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:147]:
```suggestion
`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid search term - `Texas:US`|`Austin:US`|`78740:US`. After the colon, the two-letter Country code is required to be passed in uppercase.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have made update based on your suggestion. Thanks.

**Review by BarnyShergold — APPROVED** [review]
Excellent!

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 4]
```diff
@@ -105,6 +106,23 @@ Because `pageSize` is set to 1 and distance by `distance` is applied, result wil
...
+      "total_count": 0,
+      "page_info": {
+        "page_size": 1,
+        "current_page": 1,
+        "total_pages": 0
+      }
+    }
+  }
+}
+```
```
Comment (keharper) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:125]: We don't need to show an empty response. Remove all of your additions up to here.

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 5]
```diff
@@ -125,7 +143,7 @@ Use the `AreaInput` to apply filtration by distance. All attributes are required
 Attribute | Data type | Description
 --- | --- | ---
 `radius` | Int! | The radius to the search, in kilometers.
-`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  For example, `Austin:US`.
+`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid search term - `Texas:US`|`Austin:US`|`78740:US`. After the colon, the two-letter Country code is required to be passed in uppercase.
```
Comment (keharper) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:146]: I'm not sure what you're trying to say here. Those | symbols can't be in a markdown table. 

Perhaps reword to 

Valid search terms include `Texas:US`, `Austin:US`, and `78740:US`. The two-letter country code must be uppercase

**Review by keharper — CHANGES_REQUESTED** [review]

**Comment by RakeshJesadiya** [comment]
@keharper I have updated the docs based on your suggestion. Thank you.

**Review by keharper — APPROVED** [review]

**Pull request merged**